airflow/providers/amazon/aws/hooks/dynamodb.py

```diff
@@ -18,13 +18,14 @@
 
 
 """This module contains the AWS DynamoDB hook"""
+import warnings
 from typing import Iterable, List, Optional
 
 from airflow.exceptions import AirflowException
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 
 
-class AwsDynamoDBHook(AwsBaseHook):
+class DynamoDBHook(AwsBaseHook):
```

Comment:
This require also edit to:
https://github.com/apache/airflow/blob/1baa648e2af124032c2f2752dc0f587ee253a3a9/tests/deprecated_classes.py#L415

Reply:
I'm sorry, I don't follow.   What needs to be done there?

Reply:
The entry in deprecated class points contrib to `AwsDynamoDBHook`
This is no longer true. It should point to `DynamoDBHook`